.github/.container-structure-test-config.yaml

```
@@ -18,7 +18,7 @@ commandTests:
   - name: "gcc"
     command: "gcc"
     args: ["--version"]
-    expectedOutput: ["^gcc \(Alpine 12\..*$"]
+    expectedOutput: ["^gcc \\(Alpine 12\\..*$"]
```

That's not essential though:
```suggestion
    expectedOutput: ["^gcc \\(Alpine 12\\."]
```